.github/workflows/cgroup2.yaml

```diff
@@ -0,0 +1,59 @@
+name: Cgroup v2
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  docker:
+    name: Cgroup v2
+    # nested virtualization is only available on macOS hosts
+    runs-on: macos-10.15
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        provider: [docker, podman]
+    env:
+      KIND_EXPERIMENTAL_PROVIDER: "${{ matrix.provider }}"
+      HELPER: "./hack/ci/vagrant-helper.sh"
+      JOB_NAME: "cgroup2-${{ matrix.provider }}"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Boot Fedora
+        run: |
+          ln -sf ./hack/ci/Vagrantfile ./Vagrantfile
```

Comment:
So, this runs the machine and mounts the current folder, that is the repository root, in the vagrant folder, right?

Reply:
yes